docs/input/docs/getting-started/whystrategypattern.md

```diff
@@ -0,0 +1,22 @@
+---
+Order: 10
+Title: Why does BBT.StrategyPattern exist?
+Description: Introduction to BBT.StrategyPattern, what problems are solved and how it is distinctive from other implementations of strategy pattern.
+---
+
+## Introduce structure to your code by using BBT.StrategyPattern
+
+BBT.StrategyPattern is an implementation of the strategy design pattern.
+
+BBT.StrategyPattern has the following goals:
+
+* Support the usage of strategies with no need to implement specific strategy factories
+* Provide a generic strategy provider based on generics
+* Introduce strategy categorization based on generic parameter
+
+Implementation details of BBT.StrategyPattern are:
+
+* Introduces a generic strategy interface
+* The generic parameter represents the criterion for the selection of the specific strategy
+* Works with any IoC framework
+* Supports generic type unbound IoC registrations
```

Comment:
"Generic type unbound IoC registrations" are a feature of the IoC container, not this library.